templates/tutorials/index.html

```diff
@@ -1,33 +1,41 @@
-{% extends "tutorials/base_tutorials.html" %}
-
-{% block title %}Tutorials{% endblock %}
-
-{% block content %}
-
-{% if page == 1 %}
+{% extends "tutorials/base_tutorials.html" %} {% block title %} Tutorials {%
```

Comment:
These shouldn't be collapsed into 1 line, it's much harder to read.

This will be an auto formatter of some kind. Could you disable it for HTML pages? They never do a good job with template tags.

Reply:
Yeah I noticed that, I'll try to figure out which one of my plugins is misbehaving

Reply:
I've reverted it!